### PR TITLE
Add Help section with Shortcuts button to open HC article.

### PR DESCRIPTION
**Pull request by callistusystan**
I've added the Help section at the end of the Settings section in the Side Menu.

Button links to https://support.google.com/chromebook/answer/183101#textapp

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -189,6 +189,17 @@ <h2 id="setting-theme-heading" i18n-content="themeSetting"></h2>
                 </div>
               </div>
             </div>
+            <div class="group" role="radiogroup" aria-labelledby="setting-help-heading">
```
Comment (ZufengW) [inline index.html:192]: Remove `role="radiogroup"`

File: js/controllers/menu.js [hunk 2]
```diff
@@ -131,6 +132,11 @@ MenuController.prototype.saveas_ = function() {
   return false;
 };
 
+MenuController.prototype.openShortcuts_ = function(e, tab) {
```
Comment (ZufengW) [inline js/controllers/menu.js:135]: `e, tab` looks unused. Can remove it.

**Review by ZufengW — CHANGES_REQUESTED** [review]
LGTM with nits

File: index.html (hunk 1 above)
Comment (callistusystan) [reply index.html]: Whoops, good eye! Thanks for catching this.

File: js/controllers/menu.js (hunk 2 above)
Comment (callistusystan) [reply js/controllers/menu.js]: Yeap, makes sense! Removed both arguments

**Review by ZufengW — APPROVED** [review]

**Pull request merged**